grimshot.1.scd

```diff
@@ -75,6 +75,11 @@ _window_
 _output_
 	Captures the currently active output.
 
+_anything_
+	Allows manually selection a single window (by clicking on it), an output (by
+	clicking outside of all windows, e.g. on the status bar), or an area (by
+	using click and drag).
```

Comment:
It may make sense to also add this "or an area ..." to the `window` option since it applies there too.